modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -20,6 +20,8 @@ you add a new version to an existing asset.
 
 For an HTTP API, because the asset is only an endpoint, it makes no sense to increase the patch or minor versions in every new publication. HTTP APIs therefore only have major version increases and also changes to the API version.
```

Comment:
Doesn't this sentence basically repeat the sentence in the compacted text? Consider combining them into one instead of having this separate.

Reply:
Good point. Combined.